src/ViewUser.js

```diff
@@ -525,12 +525,19 @@ class ViewUser extends React.Component {
 
   update(user) {
     const addressTypes = (this.props.parentResources.addressTypes || {}).records || [];
+    const userData = cloneDeep(user);
 
-    if (user.personal.addresses) {
-      user.personal.addresses = toUserAddresses(user.personal.addresses, addressTypes); // eslint-disable-line no-param-reassign
+    if (userData.personal.addresses) {
+      userData.personal.addresses = toUserAddresses(userData.personal.addresses, addressTypes); // eslint-disable-line no-param-reassign
     }
 
-    const { proxies, sponsors, permissions, servicePoints, preferredServicePoint } = user;
+    const {
```

Comment:
It looks like this is a good chance for us to follow our code conventions. Thanks for this update

Reply:
Agree. Done.